resources/measures/upgrade_hvac_replace_boiler_by_heatpump/measure.rb

```diff
@@ -362,9 +362,16 @@ def run(model, runner, user_arguments)
     unitarys.each do |unit|
       flowmethod = unit.supplyAirFlowRateMethodDuringHeatingOperation
       runner.registerInfo("flow method is #{flowmethod} ")
-      if flowmethod == ''
-        unit.setSupplyAirFlowRateMethodDuringHeatingOperation('SupplyAirFlowRate')
-        runner.registerInfo("SupplyAirFlowRateMethodDuringHeatingOperation is resetted to use 'SupplyAirFlowRate' method")
+      if model.version < OpenStudio::VersionString.new('3.7.0')
+        if flowmethod == ''
+          unit.setSupplyAirFlowRateMethodDuringHeatingOperation('SupplyAirFlowRate')
+          runner.registerInfo("SupplyAirFlowRateMethodDuringHeatingOperation is reset to use 'SupplyAirFlowRate' method")
+        end
+      else
+        if flowmethod == ''
+          unit.autosizeSupplyAirFlowRateDuringHeatingOperation
+          runner.registerInfo("Utilized autosizeSupplyAirFlowRateMethodDuringHeatingOperation method")
```

Comment:
As of 3.7.0, the `[autosizeSupplyAirFlowRateDuringHeatingOperation](https://s3.amazonaws.com/openstudio-sdk-documentation/cpp/OpenStudio-3.7.0-doc/model/html/classopenstudio_1_1model_1_1_air_loop_h_v_a_c_unitary_system.html#a50cb4fa8134d6e77e500dc33fd6a806a)' method will will set the flow "[method](https://bigladdersoftware.com/epx/docs/23-2/input-output-reference/group-unitary-equipment.html#field-heating-supply-air-flow-rate-method-000)" to 'SupplyAirFlowRate', so the original info message is still correct and should be kept.

Reply:
Updated to use the original text